EECS3311-PD1/src/main/java/Analyzer/AirPollutionvsForestArea.java

```diff
@@ -0,0 +1,35 @@
+package Analyzer;
+
+import Fetchers.AbstractFetcher;
+import Fetchers.AirPollutionFetcher;
+import Fetchers.ForestAreaFetcher;
+import org.example.WbApiModel;
+
+import java.util.Map;
+
+public class AirPollutionvsForestArea extends AbstractAnalyzer{
+    private Map<String, Double> AirPollutionData;
+    private Map<String, Double> ForestAreaData;
+
+    public AirPollutionvsForestArea(String sYear, String eYear, String country) {
+        super(sYear, eYear, country);
+    }
+
+    public AnalysisResult recalculate() throws WbApiModel.WbApiModelException {
+        String[] labels = new String[2];
+        AbstractFetcher absFetch = new AirPollutionFetcher(this.sYear, this.eYear, this.country);
+        this.AirPollutionData = absFetch.getData();
+        labels[0] = absFetch.getLabel();
```

Comment:
I only realized now, this can be entirely replaced with `fetcher.getClass().getName()`
And then we won't need to have change Fetchers to implement `getLabel()`